CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ Remember to align the itemized text with the first line of an item within a list
 ## jax 0.4.30
 
 * Changes
+  * `jax.numpy.ceil`, `jax.numpy.floor` and `jax.numpy.trunc` are now preserving
+    input integral dtypes.
```

Comment:
JAX 0.4.30 has already been released – this note needs to go under jax 0.4.31.

Also, consider rephrasing this for clarity. I'm not sure that most users will understand "now preserving input integral types". Maybe say they "no longer upcast integer inputs to floating point"

Reply:
Thanks, haven't spotted the new release, sorry. How about this: 
```
## jax 0.4.31

* Changes
  * `jax.numpy.ceil`, `jax.numpy.floor` and `jax.numpy.trunc` now return the output
  of the same dtype as the input, i.e. no longer upcasting integer inputs to floating
  point.

```